docs/user-guide.md

```diff
@@ -0,0 +1,856 @@
+    improve.
+
+This guide has three aims:
+
+1. A clear overview, context setting and sign-posting for all things PyScript.
+2. Exploration of PyScript in substantial technical detail.
+3. Demonstration of the features of PyScript working together in real-world
+   example applications.
+
+_Read this page in full_: you will acquire a comprehensive overview of the
+PyScript platform along with suggestions for where next to explore and learn.
+
+For more dynamic support, join in the PyScript conversation on our
+[discord server](https://discord.gg/HxvBtukrg2). There you'll find core
+developers, community contributors and a flourishing forum for those creating
+projects with PyScript. Should you wish to engage with the development of
+PyScript, you are welcome to engage and make contributions via
+[the project's GitHub organisation](https://github.com/pyscript).
+
+Finally, the examples listed at the end of this page are all freely available
+and copiously commented on [pyscript.com](pyscript.com).
+
+!!! note
+
+    Many of these examples come from contributors in our wonderful
+    community. If you believe you have a project that would make a good
+    demonstration example, please don't hesitate to
+    [get in touch](https://discord.gg/HxvBtukrg2).
+
+## What is PyScript?
+
+[PyScript](pyscript.net) is a platform for [Python](python.org) in the
+[browser](https://en.wikipedia.org/wiki/Web_browser).
+
+PyScript brings together two of the most vibrant technical ecosystems on the
+planet. If [the web](https://en.wikipedia.org/wiki/World_Wide_Web) and Python
+had a baby, you'd get PyScript.
+
+PyScript works because modern browsers support
+[web assembly](https://webassembly.org/) (abbreviated to WASM) - an
+[instruction set](https://en.wikipedia.org/wiki/Instruction_set_architecture)
+for a [virtual machine](https://en.wikipedia.org/wiki/Virtual_machine) with
+an open specification and near native performance. PyScript takes
+versions of the Python interpreter compiled to WASM, and makes them easy to use
+inside the browser.
+
+At the core of PyScript is a _philosophy of digital empowerment_. The web is
+the world's most ubiquitous computing platform, mature and familiar to billions
+of people. Python is one of the
+[world's most popular programming languages](https://spectrum.ieee.org/the-top-programming-languages-2023):
+it is easy to teach and learn, used in a plethora of existing domains
+(such as data science, games, embedded systems, artificial intelligence,
+finance, physics  and film production - to name but a few), and the Python
+ecosystem contains a huge number of popular and powerful libraries to address
+its many uses.
+
+PyScript brings together the ubiquity, familiarity and accessibility of the web
+with the power, depth and expressiveness of Python. It means PyScript isn't
+just for programming experts but, as we like to say, for the 99% of the rest of
+the planet who use computers.
+
```

Comment:
is this blurb actually needed in a "User Guide"?
I suspect that people coming to read the user guide are already convinced to use pyscript.
Maybe it should to into an `Overview` or `Motivation` page?
(no need to split it now, we can do it later)

Reply:
Yes it does... (Fabio and I discussed this and the "features" section too). Folks will be directly linked to this page as the user guide... so likely miss the intro blurb.